pkg/lb/forwarder/endpointSlice.go

```diff
@@ -0,0 +1,116 @@
+package forwarder
+
+import (
+	"fmt"
+
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	discoveryv1 "k8s.io/api/discovery/v1beta1"
+
+	lbv1 "github.com/harvester/harvester-load-balancer/pkg/apis/loadbalancer.harvesterhci.io/v1alpha1"
+	ctlDiscoveryv1 "github.com/harvester/harvester-load-balancer/pkg/generated/controllers/discovery.k8s.io/v1beta1"
+)
+
+const (
+	KeyLabelForwarder = "loadbalancer.harvesterhci.io/forwarder"
+	KeyServiceName    = "kubernetes.io/service-name"
```

Comment:
Is this a known label key?

Reply:
Yes.

Reply:
Do you have reference in K8s docs

Reply:
https://kubernetes.io/docs/concepts/services-networking/endpoint-slices/#ownership